packages/api/prisma/schema.prisma

```diff
@@ -8,16 +8,16 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
+  id       String @id @default(uuid())
```

Comment:
I don't know Prisma. 
But do you think about @db.Uuid? Is it required / useful? 
Because as I see in migration.sql there is `TEXT`, whereas postgres have UUID type. I'd be great to use it for better indexing.

![image](https://user-images.githubusercontent.com/18637116/129957426-69e69054-3e31-4127-882c-bce5db9b5e1d.png)